### ISSUE TITLE
Fix Create Course Layout

### ISSUE THREAD

**Issue by anand-harsh** [anand-harsh/Edumi#9]
**Issue:** Fix the layout of Create course page (/admin/createcourse) .The Sidebar for the dashboard should be on right hand side.

**Comment by Vansh-Narang** [comment]
Wanna work on it !

**Comment by anand-harsh** [comment]
Sure @Vansh-Narang , I will assign you this issue.

**Comment by Vansh-Narang** [comment]
![Screenshot (455)](https://github.com/anand-harsh/Edumi/assets/102366482/041ff3f7-db95-46a2-a0e3-02bdea301981)

is this not done?

**Comment by anand-harsh** [comment]
> ![Screenshot (455)](https://user-images.githubusercontent.com/102366482/277359542-041ff3f7-db95-46a2-a0e3-02bdea301981.png)
> 
> is this not done?

Yes it is fixed in other pages, but once check " /admin/createcourse ". This need to be fixed.
